learning/02_graphics/how_to_use_glm.markdown

```diff
@@ -0,0 +1,121 @@
+---
+.. title: The new GLM syntax 
+.. summary: How to use glm vectors instead the legacy ones
+.. type: howto
+---
+
+*Note!*: This page is useful only if you are working with the current master branch. You can skip this information if you are working with the version 0.9.3 that you have downloaded from this website.
+
+The next version of openFrameworks will replace the internal math library with [GLM](http://glm.g-truc.net). GLM is a solid C++ library used for all the math operations needed when doing vectors and matrices operations. The use of this library implies some change in the syntax used to declare vectors and to execute vector's operation. The legacy mode is still supported, but the new mode, enabled by default, uses the new glm syntax.
+
+If you are not interested using this library and you want to continue using the syntax you were used to, you can easily disable it using the OF_USE_LEGACY_MESH constant in ofConstants.h
+
```

Comment:
OF_USE_LEGACY_MESH doesn't really disable glm completely only for containers (ofMesh and ofPolyline) for simple objects like return values or parameters ofVec* auto converts from/to glm equivalents so it's not so necessary but there's no way to auto convert a container of an object into a container of other and that's why we have this macro